package.json

```diff
@@ -94,6 +95,7 @@
     "react-native-extended-stylesheet": "^0.12.0",
     "react-native-fingerprint-scanner": "git+https://github.com/hieuvp/react-native-fingerprint-scanner.git#9cecc0db326471c571553ea85f7c016fee2f803d",
     "react-native-gesture-handler": "^2.9.0",
+    "react-native-gradle-plugin": "^0.71.15",
```

Comment:
this should not be needed? what platform are you using?

in any case, if needed, it should be a dev-dependancies not a dependancy

Reply:
ok i will remove it